symfony/plugins/orangehrmMarketPlacePlugin/web/js/ohrmAddonSuccess.js

```diff
@@ -33,6 +33,21 @@ $(document).ready(function () {
     });
     $('.installBtn').live('click', function (event) {
         installId = $(this).attr('addid');
+
+        $.ajax({
+            method: "POST",
+            data: {addonID: installId},
+            url: prerequisiteVerificationUrl,
```

Comment:
What is this prerequisiteVerificationUrl ? is it MP endpoint ? if so you can't call cross domain ajax requests ..  should be handled via php guzzle GET

Reply:
No Samantha, It is an opensource action. Anyway Damith told that this is not a must to convert it into a get request. That's why I kept it as it is.